Assets/SEE/Utils/VersionControlSystems.cs

```diff
@@ -0,0 +1,144 @@
+using UnityEngine;
+using LibGit2Sharp;
+using System.Linq;
+using System.IO;
+
+/// <summary>
+/// Contains all version control systems as subclasses, which derives from IVersionControl,
+/// so that all have the same functionality.
+/// </summary>
+public class VersionControlSystems : MonoBehaviour
+{
+    /// <summary>
+    /// Implements the functionality of IVersionControl for Git.
+    /// </summary>
+    public class GitVersionControl : IVersionControl
+    {
+        /// <summary>
+        /// The name of the file, when it got deleted, or renamed.
+        /// </summary>
+        private string name = null;
```

Comment:
That is unclear to me. A VersionControl has a single file as an attribute? To me, this attribute looks more like a local variable of any of the methods contained in this class.